static/js/tools/stat_var/explorer.tsx

```diff
@@ -23,74 +25,89 @@ interface ExplorerPropType {
   summary: StatVarSummary;
 }
 
-interface PlaceTypeSummaryListElement {
-  listElement: HTMLElement;
-  numPlaces: number;
-  placeType: string;
-}
-
 class Explorer extends Component<ExplorerPropType, unknown> {
   render(): JSX.Element {
-    const placeTypeSummaryList = this.createPlaceTypeSummaryList();
+    const provenanceSummaryList = this.flattenProvenanceSummary();
     return (
-      <div id="placeholder-container">
-        <h1 className="mb-4">{this.props.displayName}</h1>
-        <p>
-          dcid: {this.props.statVar}
-          <br></br>
-          <a href={`/browser/${this.props.statVar}`}>Graph Browser node</a>
-        </p>
-        This statistical variable has observations for the following places:
-        <ul>
-          {placeTypeSummaryList.map((element) => {
-            return element.listElement;
-          })}
-        </ul>
+      <div id="stat-var-explorer">
+        <h1>{this.props.displayName}</h1>
+        <h2>
+          dcid:{" "}
+          <a
+            href={`/browser/${this.props.statVar}`}
+            target="_blank"
+            rel="noreferrer"
+          >
+            {this.props.statVar}
+          </a>
+        </h2>
+        {!this.props.summary && <div>No data available.</div>}
+        {this.props.summary?.placeTypeSummary && (
+          <h4 className="highlight-text">
+            Total number of places: {this.getNumberOfPlaces()}
+          </h4>
+        )}
+        {this.props.summary?.provenanceSummary && (
+          <h4 className="highlight-text">
+            Total number of sources: {provenanceSummaryList.length}
+          </h4>
+        )}
+        {this.props.summary?.placeTypeSummary && (
+          <div id="place-type-summary-section" className="browser-page-section">
+            <h3>Places</h3>
+            <Place
```

Comment:
should the component name also suggest that it also contains multiple places or place examples?

Reply:
good idea! changed to "Places"